app/models/lunch.rb

```diff
@@ -62,17 +64,14 @@ def einstein
     end
 
     def chalmrest
-      date = Time.new.strftime("%Y-%m-%d")
-      locale = I18n.locale.to_s
```

Comment:
Is `date = Time.new.strftime("%Y-%m-%d")` used for anything i Rails by the way? That variable didn't seem to be used.